src/llama-sampling.cpp

```diff
@@ -239,7 +240,7 @@ static void llama_sampler_top_k_impl(llama_token_data_array * cur_p, int32_t k)
 
     // Sort scores in descending order
     if (!cur_p->sorted) {
-        auto comp = [](const llama_token_data & a, const llama_token_data & b) {
+        auto comp = [](const llama_token_data a, const llama_token_data b) {
```

Comment:
Remove this change

Reply:
fix it https://github.com/ggerganov/llama.cpp/pull/11294/commits/44ec40a43a3b6302a50aa934d8148cb8f5c11b71